hr_employee_calendar_planning_customize/models/hr_employee.py

```diff
@@ -50,8 +49,11 @@ def _compute_antiquity(self):
             years = days // 365
             days = days % 365
 
-            rec.antiquity_years = years
             rec.antiquity = days
+            if years == 0 and days == 0:
```

Comment:
Podrías probar con :
`if not years and not days:`
Creo que sería lo mismo porque 0 entería que es False. pruebalo y me dices